test/Bond.spec.ts

```diff
@@ -405,6 +405,36 @@ describe.only("Bond", () => {
               });
             });
 
+            it("allows a partial withdraw", async () => {
+              const owed = await bond.amountOwed();
+              await paymentToken.approve(bond.address, owed);
+              await bond.pay(owed);
+
+              await expectTokenDelta(
+                async () =>
+                  bond.withdrawExcessCollateral(
+                    (await bond.previewWithdraw(0)).div(2),
+                    owner.address
+                  ),
+                collateralToken,
+                owner,
+                bond.address,
+                config.collateralTokenAmount.div(2)
+              );
+            });
+            it.only("fails if requesting too much", async () => {
```

Comment:
remove the `only` I would suggest using the 
`npx hardhat test --grep "requesting too"` 
commands. https://ethereum.stackexchange.com/a/123403/93861

Reply:
That's cool. I usually use the mocha extension though. Removed